workerapi/server/server.go

```diff
@@ -59,14 +59,15 @@ func (h *handler) stats() {
 					numActive++
 				}
 			}
+			lastContact := int64(time.Now().Sub(h.timeLastRpc) / time.Millisecond)
```

Comment:
lastContact sounds like a Time, not a duration; maybe something like timeSinceLastContact or staleness? Loneliness?